book-shop/go.mod

```diff
@@ -0,0 +1,91 @@
+module github.com/cloudwego/biz-demo/book-shop
+
+go 1.18
+
+require (
+	github.com/apache/thrift v0.13.0
+	github.com/cloudwego/kitex v0.4.3
+	github.com/hertz-contrib/swagger v0.0.0-20220711030440-b6402d4709f0
+	github.com/swaggo/files v0.0.0-20220728132757-551d4a08d97a
+)
+
+require (
+	github.com/bwmarrin/snowflake v0.3.0
```

Comment:
I think we could use more extensions to improve the system such as otel、gzip、pprof